tools/cockpit.spec

```diff
@@ -450,15 +455,14 @@ getent passwd cockpit-wsinstance >/dev/null || useradd -r -g cockpit-wsinstance
 # firewalld only partially picks up changes to its services files without this
 test -f %{_bindir}/firewall-cmd && firewall-cmd --reload --quiet || true
 
-%if 0%{?rhel} || 0%{?fedora} == 29
+%if 0%{?rhel}
```

Comment:
Forgot to mention that in the commit message (happy to do on squashing): F29 is EOL now, and we stopped uploading to F29 some months ago already.